syntax/sexplib.vim

```diff
@@ -3,9 +3,9 @@
 " Filenames:    *.sexp
 " Maintainers:  Markus Mottl      <markus.mottl@gmail.com>
 " URL:          http://www.ocaml.info/vim/syntax/sexplib.vim
-" Last Change:  2012 Jun 20 - Fixed a block comment highlighting bug (MM)
+" Last Change:  2017 Apr 11 - Improved matching of negative numbers (MM)
+"               2012 Jun 20 - Fixed a block comment highlighting bug (MM)
 "               2012 Apr 24 - Added support for new comment styles (MM)
-"               2009 Apr 02 - First release (MM)
```

Comment:
Was the removal of this change note intentional?

Reply:
Indeed, I tried to stick to the convention I inferred (even though I don’t like it so much).